Language/Functions/External_Functions/not_connected_to_a_body_t01.dart

```diff
@@ -22,9 +20,11 @@ import '../../../Utils/expect.dart';
 external f();
 external g([var x]);
 external h({var x});
+external m({required var x});
 
 main() {
   Expect.throws(() { f(); }, (e) => e is NoSuchMethodError);
   Expect.throws(() { g(); }, (e) => e is NoSuchMethodError);
   Expect.throws(() { h(); }, (e) => e is NoSuchMethodError);
+  Expect.throws(() { m(x: ""); }, (e) => e is NoSuchMethodError);
```

Comment:
Would it work to expect a web error here using `[web]` (works like `[cfe]` and `[analyzer]`?

https://github.com/dart-lang/sdk/wiki/Testing#web-static-errors

Otherwise splitting and annotating with status files SGTM as well. Requirements don't support this, yet.

Reply:
Ah, interesting! I forgot about those `[web]` expectations.

So if non-`[web]` configurations will ignore the compile-time errors for the `[web]`, we should be OK!

One way to ensure this would be to temporarily have an unavoidable run time error (like `throw 0;` in main) and then confirm that we get a test failure on at least some configurations where the test is executed.

(Otherwise, a test  that expects a compile-time error is just ignored by every configuration that runs the program).

Reply:
> Would it work to expect a web error here using [web] (works like [cfe] and [analyzer]?

It will work but in this case these tests won't be run on `vm_...` platforms. Test that expecting compile-time errors are not run at runtime (even if we are expecting error on web-platform only). So, I think, updating a status file is the only choice.

Reply:
> So if non-[web] configurations will ignore the compile-time errors for the [web], we should be OK!

Unfortunately all runtime configuration will ignore this tests as well. If some test has
```dart
//  ^
// [any platform]
```
then this test is skipped by all configurations which actually run the tests. We'll have to update status file to exclude these tests from web-platforms

Reply:
So you're saying that it is known that even though the compile-time error expectation is for `[web]`, the test will be skipped on vm configurations? That's what I suspected, but did not know. In any case, status files would be safer.

So lets do that!

Reply:
I'll do that together with the next roll

Reply:
OK, let me see if I've got this right: This PR can be landed now, even though it does not work on web configurations, because there will be some other PR (on the SDK, presumably) that changes the status files such that these tests won't be enabled for web platforms.

But then we'd have yet another PR (on the co19 repo) adding tests which are similar to these, but where the _declarations_ of external functions are expected to give rise to a compile-time error (and `main` wouldn't do anything, it would just mention stuff that shouldn't be tree shaken out), and they will be enabled on web platforms only, again by the status files.

Is that in line with your plans?

Reply:
No. I'm going only to turn-off tests that fail on web-platforms. I'm not going to add any tests expecting compile errors on  web. Specification doesn't state that we should have a compile-time error in case of external functions on web-platforms. So my plan is to merge this PR. Then, when doing a roll that enables this change also change a status file. That's it